### ISSUE TITLE
failed to install modpack due to an unknown error

### ISSUE THREAD

**Issue by DoubleVision50** [FTBTeam/FTB-App#1068]
### What Operating System

Windows 11

### Debug Code

FTB-DBGOQIRORUVEZ

### Describe the bug

if i go to install any modpack at all, i get the error message stated. i have uninstalled and reinstalled the app multiple times, and yet the problem persists.  i had been playing stoneblock with friends with no issue, and suddenly it wasnt in my library anymore and all the stated issues started happening.

### Steps to reproduce

go to any modpack, click install, and as soon as it starts to prepare intstall, it fails and gives error message

### Expected behaviour

modpack would install and appear in library

**Comment by Aipoc76** [comment]
I cant download Stone block 3.  gives same error.    Failed to install Stoneblock 3 due to a unknown error.

**Comment by NymArchon** [comment]
I also have this error, i cannot dl any packs. the ftb ap gives me the failed to install due to unknown error and windows says a java exception has occurred 
![Screenshot 2024-02-11 134529](https://github.com/FTBTeam/FTB-App/assets/159729122/1fde6535-522a-4520-8a9b-72b225a7b7b3)

**Comment by WoutV01** [comment]
I have the same error, for any modpack

**Comment by MichaelHillcox** [comment]
Could you confirm if this happens in the latest version of the app?

**Comment by Aipoc76** [comment]
same error.
![image](https://github.com/FTBTeam/FTB-App/assets/42475106/2dccbaf5-8365-4720-90e4-09d713e6bfdc)

I did however notice in the log :
Cannot run program "C:\Users\######\AppData\Local\

where the  ##### is the wrong username.  What file would i correct that under the FTB app?

**Comment by MichaelHillcox** [comment]
Did you at some point move files from one place to another?

**Comment by Aipoc76** [comment]
> Did you at some point move files from one place to another?

yes, changed username, but that was years ago and before FTB ever installed and there have been many windows updates / version installs since  then.

**Comment by Starrmannn** [comment]
had this exact issue and it was because I had also changed my username on windows (bad idea in hindsight). was able to fix it by going to (C:)\users\(old username) and deleting the folder named .ftb

not sure how helpful that advice actually is, but I figured I'd share since I couldn't find any fixes for this issue around the internet, hopefully it at least sets someone on the right path

**Comment by CylorB** [comment]
> had this exact issue and it was because I had also changed my username on windows (bad idea in hindsight). was able to fix it by going to (C:)\users(old username) and deleting the folder named .ftb
> 
> not sure how helpful that advice actually is, but I figured I'd share since I couldn't find any fixes for this issue around the internet, hopefully it at least sets someone on the right path. It deletes all your modpacks when you do this by the way, so make sure you write down all their names somewhere before doing this if you want to keep all of them.

This worked for me good stuff man
